ramls/event.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Event data model",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "description": "UUID",
+      "type": "string"
```

Comment:
Typically we constrain UUID fields using a [regular expression](https://dev.folio.org/guides/uuids/)

Reply:
Thanks, I'll add the constraint.